src/kixi/gwyn/gwyn.clj

```diff
@@ -47,20 +46,16 @@
 (defn get-env-var [key]
   (get (System/getenv) "places_api_key"))
 
-(defn fetch-url [url] ;;; switch to slurp
-  (html/html-resource (java.net.URL. url)))
-
 (defn api-str [loc k]
   (str google-places-api loc "&key=" (get-env-var k)))
 
+(defn fetch-url [api-call]
+  (try (slurp api-call)
+       (catch Exception e (str "Fire station does not exist. Incorrect URL"))))
```

Comment:
Returning a string here will break your `parse-google-places` code. Either re-throw with your own error msg (making sure to include the original exception) or adjust your `parse-google-places` fn.